Substitutions: script output and human revision (#1363)
codenvy/che-docs: src/main/pages/che-7/overview/ref_hosted-che-faq-and-troubleshooting.adoc

// hosted-{prod-id-short}
:page-liquid:

> [id="hosted-{prod-id-short}-faq-and-troubleshooting_{context}"]

Reviewer: ```suggestion
[id="hosted-che-faq-and-troubleshooting_{context}"]
```

= Frequently Asked Questions (FAQs) and Troubleshooting

== FAQs

> **Q**: Where can I see the up-to-date status of the {prod} hosted by Red Hat?

Reviewer: ```suggestion
**Q**: Where can I see the up-to-date status of Eclipse Che hosted by Red Hat?
```

**A**: There is a dedicated status page - https://che.statuspage.io/

== Troubleshooting

> === Cannot log into the Hosted {prod-short} - Authorization token is missed

Reviewer: ```suggestion
=== Cannot log into the Hosted Che - Authorization token is missed
```

Click here to reload page.
----

> Also, note that telemetry is enabled in Hosted {prod-short} so Woopra / Segment tracking scripts must be explicitly allowed in case being blocked by a browser extension:

Reviewer: ```suggestion
Also, note that telemetry is enabled in Hosted Che, so Woopra or Segment tracking scripts must be explicitly allowed if blocked by a browser extension:
```
